Tiltfile

```diff
@@ -21,6 +21,7 @@ settings = {
     "preload_images_for_kind": True,
     "kind_cluster_name": "capz",
     "capi_version": "v1.6.3",
+    "caaph_version": "v0.1.1-alpha.1",
```

Comment:
should we update to the latest release? (congrats btw!)

Reply:
The latest release (v0.2.0) is broken, so I think we should go with this as the latest working version.

Reply:
@jackfrancis We can also hold this for the upcoming v0.2.1 release as well. WDYT?

Reply:
Let's bump this to the [v0.2.1](https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/releases/tag/v0.2.1) release.